magnum_capi_helm/driver_utils.py

```diff
@@ -34,7 +34,7 @@ def sanitized_name(name, suffix=None):
         "[^a-z0-9]+",
         "-",
         (f"{name}-{suffix}" if suffix else name).lower(),
-    )
+    ).strip("-")
```

Comment:
Why is this needed?

Reply:
subdomains have to start and end with alphanumerics, so a name like `__DEFAULT__` would be `--default--` without this